packages/venia-ui/lib/components/AddressBookPage/addressCard.css

```diff
@@ -5,13 +5,43 @@
     display: grid;
     grid-template-columns: 1fr max-content;
     padding: 1.25rem 2rem;
+    position: relative;
 }
 
 .root_updated {
     composes: root;
     animation: flash var(--venia-global-anim-bounce) 640ms 2;
 }
 
+.confirmDeleteContainer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(255, 255, 255, 0.9);
+    display: grid;
+    justify-items: center;
+    align-items: center;
+    grid-template-columns: 1fr 1fr;
+    gap: 0.5rem;
+    padding: 1rem;
+}
+
+.confirmDeleteButton {
+    composes: root_normalPriorityNegative from '../Button/button.css';
+
+    background-color: white;
+    max-width: 6rem;
```

Comment:
It'd be great to use a token for the color. 
As for the `max-width`, this is very specific to how the English words look. We may not want this in the end.